frontend/src/components/readsets/ReadsetsListContent.tsx

```diff
@@ -0,0 +1,256 @@
+            <FilterPanel
+                descriptions={[]}
+                filters={filters}
+                setFilter={readsetTableCallbacks.setFilterCallback}
+                setFilterOption={readsetTableCallbacks.setFilterOptionsCallback}
+            />
+            <div style={{
+                display: 'flex',
+                justifyContent: 'space-between'
+            }}>
+                {extraButtons}
+                <FiltersBar filters={filters} clearFilters={readsetTableCallbacks.clearFiltersCallback} />
+            </div>
+            <PagedItemsTable<ObjectWithReadset>
+                columns={columns}
+                expandable={{
+                    columnTitle: () => <div>View Metrics</div>,
+                    expandIcon: ({ expanded, onExpand, record }) =>
+                        expanded ? (
+                            <Tooltip title="Hide Metrics">
+                                <MinusCircleTwoTone onClick={e => onExpand(record, e)} />
+                            </Tooltip>
+                        ) : (
+                            <Tooltip title="View Metrics">
+                                <PlusCircleTwoTone onClick={e => onExpand(record, e)} />
+                            </Tooltip>
+
+                        )
+                    ,
+                    expandedRowRender: (record) => {
+                        const readset: Readset = record.readset
+                        return (
+                            <div style={{
+                                display: 'grid',
+                                gridTemplateColumns: 'repeat(8,1fr)',
+                                gap: '1em'
+                            }} key={readset.id}>
+                                {
+                                    readset.metrics ?
+                                        Object.keys(readset.metrics).map(
+                                            (name) => {
+                                                return (
+                                                    readset.metrics && (readset.metrics[name].value_numeric || readset.metrics[name].value_string) &&
+
+
+                                                    <div style={{
+                                                        display: 'flex',
+                                                        flexDirection: 'column',
+                                                    }} key={name}>
+                                                        {<b >
+                                                            {name.replace(/_/g, " ")}
+                                                        </b>
+                                                        }
+                                                        {readset.metrics[name].value_numeric
+                                                            ?
+                                                            Number(readset.metrics[name].value_numeric).toFixed(3)
```

Comment:
pf_clusters, yield, nb_reads, nb_bases are integer numbers. Maybe make a small helper to strip precision numbers if they are 0.